src/alt.js

```diff
@@ -34,6 +34,13 @@ function uid(container, name) {
   return key
 }
 
+/* for now just detect whether it's instance of Immutable.js */
+function isImmutable(state) {
```

Comment:
Agree with you and I'm waiting for answer here:
https://github.com/facebook/immutable-js/issues/421